qbft/types.go

```diff
@@ -27,6 +27,10 @@ type Syncer interface {
 	SyncDecidedByRange(identifier types.MessageID, from, to Height)
 }
 
+type RootGetter interface {
+	GetRoot() ([]byte, error)
+}
```

Comment:
As we discussed.. this may later be moved outside of qbft

Reply:
it already exists in types